jobs-builder/jobs/include/os2node.yaml.inc

```diff
@@ -0,0 +1,21 @@
+{#
+# Copyright (c) 2020 Red Hat, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Convert OS name to node label string.
+#}
+{% if os == "centos8" -%}
+centos8_azure
+{% elif os == "debian10" -%}
+debian10_azure
+{% elif os == "fedora32" -%}
+fedora32_azure
+{% elif os == "opensuse15" -%}
+opensuse15-azure
+{% elif os == "rhel8" -%}
+rhel8_azure
+{% elif os == "ubuntu1804" -%}
+ubuntu1804_azure
```

Comment:
I think we're mostly using ubuntu2004 nowadays

Reply:
The firecracker jobs http://jenkins.katacontainers.io/view/PR%20Triggered/job/kata-containers-2.0-firecracker-ubuntu-PR , http://jenkins.katacontainers.io/job/kata-containers-2.0-ubuntu-18.04-FIRECRACKER-main-baseline and http://jenkins.katacontainers.io/view/PR%20Triggered/job/kata-containers-2.0-tests-firecracker-ubuntu-PR are still using Ubuntu 18.04. I will keep that configuration, we can delete it in a follow up if needed.